sal_interfaces/Electrometer/Electrometer_Events.xml

```diff
@@ -84,28 +86,29 @@
 <SALEvent>
     <Subsystem>Electrometer</Subsystem>
     <Version>3.8</Version>
-    <Author></Author>
+    <Author>aanania</Author>
     <EFDB_Topic>Electrometer_logevent_intensity</EFDB_Topic>
     <Alias>intensity</Alias>
+	<Description>Intensity of light measured with the photo-diode in the unit previously configured</Description>
     <Explanation>http://sal.lsst.org</Explanation>
     <item>
         <EFDB_Name>intensity</EFDB_Name>
-        <Description>Measure intensity</Description>
+        <Description>Measure read by the device</Description>
         <IDL_Type>double</IDL_Type>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
         <EFDB_Name>unit</EFDB_Name>
-        <Description></Description>
+        <Description>Unit used </Description>
         <IDL_Type>string</IDL_Type>
         <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
         <EFDB_Name>timestamp</EFDB_Name>
-        <Description></Description>
+        <Description>Timestamp when the measure was taken</Description>
```

Comment:
measurement

Reply:
OK, but i just meant replace the word measure with measurement... just grammar.

Reply:
Ok